src/app/modules/navbar/navbar.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="navbar-index" *ngIf ="currentUrl !== 'auth'" [ngClass]="currentUrl == 'landing' ? 'blue-nav' : 'white-nav'">
+  <nav class="navbar-space row no-gutters">
+    <div class="navbar-space__left col-6">
+      <div class="logo-space" [routerLink]="['/']">
+        <img class="logo" src='../../../../assets/images/brand-logo/brand-logo.svg' />
+      </div>
```

Comment:
@Nishanthh I believe directly using `/assets/images/brand-logo/brand-logo.svg` will work, can you try it?